### PR TITLE
[wxWidgets] Fix issue with port usage

### PR DESCRIPTION
**Pull request by AenBleidd**
Fixed issue when necessary header file was not installed that leaded to this error:
`\vcpkg\installed\x64-windows-static\include\wx\platform.h(160,10): fatal error C1083: Cannot open include file: 'wx/setup.h': No such file or directory`

Also fixed string replace enumeration to include only *.h files because otherwise for some reason *.cur file was messed up.

Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

File: ports/wxwidgets/portfile.cmake [hunk 1]
```diff
@@ -66,13 +66,17 @@ endif()
 # do the copy pdbs now after the dlls got moved to the expected /bin folder above
 vcpkg_copy_pdbs()
 
+if(EXISTS ${CURRENT_PACKAGES_DIR}/lib/mswu/wx/setup.h)
+    file(RENAME ${CURRENT_PACKAGES_DIR}/lib/mswu/wx/setup.h ${CURRENT_PACKAGES_DIR}/include/wx/setup.h)
```
Comment (ras0219) [inline ports/wxwidgets/portfile.cmake:70]: Should this instead be
```cmake
file(WRITE "${CURRENT_PACKAGES_DIR}/include/wx/setup.h" "#include \"../../lib/mswu/wx/setup.h\"")
```

Comment (AenBleidd) [reply ports/wxwidgets/portfile.cmake]: @ras0219, fixed

**Comment by AenBleidd** [comment]
OSX CI build failed with no logs. Some infrastructure issue?

**Comment by JackBoosY** [comment]
```
CMake Error at ports/wxwidgets/portfile.cmake:75 (file):
  file failed to open for reading (No such file or directory):

    /Users/vagrant/Data/packages/wxwidgets_x64-osx/lib/mswu/wx/setup.h

```

**Comment by AenBleidd** [comment]
@JackBoosY, fixed

**Comment by ras0219-msft** [comment]
This LGTM, thanks for the fix and addressing my comment :)

**Pull request merged**